tools/container/toolkit/toolkit.go

```diff
@@ -478,14 +549,17 @@ func installLibrary(libName string, toolkitRoot string) error {
 
 // installToolkitConfig installs the config file for the NVIDIA container toolkit ensuring
 // that the settings are updated to match the desired install and nvidia driver directories.
-func installToolkitConfig(c *cli.Context, toolkitConfigPath string, nvidiaContainerCliExecutablePath string, nvidiaCTKPath string, nvidaContainerRuntimeHookPath string, opts *options) error {
+func installToolkitConfig(c *cli.Context, toolkitConfigPath string, nvidiaContainerCliExecutablePath string, nvidiaCTKPath string,
+	nvidaContainerRuntimeHookPath string, runtimeBinaryPaths []string, opts *options) error {
 	log.Infof("Installing NVIDIA container toolkit config '%v'", toolkitConfigPath)
 
 	cfg, err := loadConfig(nvidiaContainerToolkitConfigSource)
```

Comment:
Instead of creating the default config below and then overriding it here, we should probably rather switch this to:
```suggestion
	cfgToml, err := config.New(
		config.WithConfigFile(opts.Config),
	)
	if err != nil {
		return fmt.Errorf("unable to create config: %v", err)
	}
```
as is done here https://github.com/NVIDIA/nvidia-container-toolkit/blob/9b72161b63645eca35fbec9622efc680993e8427/cmd/nvidia-ctk/config/config.go#L122

We would then have a valid default config.